Flexible Learning/cli/notifyStaff.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/*
+Gibbon, Flexible & Open School System
+Copyright (C) 2010, Ross Parker
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+use Gibbon\Services\Format;
+use Gibbon\Contracts\Comms\Mailer;
+use Gibbon\Comms\NotificationSender;
+use Gibbon\Domain\Staff\StaffGateway;
+use Gibbon\Module\FlexibleLearning\Domain\UnitSubmissionGateway;
+
+$_POST['address'] = '/modules/Flexible Learning/report_workPendingFeedback.php';
+
+require __DIR__.'/../../../gibbon.php';
+
+// Setup some of the globals
+getSystemSettings($guid, $connection2);
+setCurrentSchoolYear($guid, $connection2);
+Format::setupFromSession($container->get('session'));
+
+if (!isCommandLineInterface()) {
+    echo __('This script cannot be run from a browser, only via CLI.');
+    return;
+}
+
+if (isSchoolOpen($guid, date('Y-m-d'), $connection2, true) == false) { 
+    echo __('School is not open, so no emails will be sent.');
+    return;
+}
+
+if ($_SESSION[$guid]['organisationEmail'] == '') {
+    echo __('This script cannot be run, as no school email address has been set.');
+    return;
+}
+
```

Comment:
Should there be a final check for the feedback mode: I think we only want this to send at times when expectFeedback == "Y". What do you think?

Reply:
Right! Yep, I'll add this.